README.md

```diff
@@ -77,55 +77,60 @@ COMMANDS
 ```
 $ kvm-install-vm help create
 NAME
-    kvm-install-vm create [OPTIONS] VMNAME
+    kvm-install-vm create [COMMANDS] [OPTIONS] VMNAME
```

Comment:
Isn't `create` one of the `COMMANDS`?

Reply:
Seems so. This was just the output i got after running `kvm-install-vm help create`

```
printf "    $prog create [COMMANDS] [OPTIONS] VMNAME\n"
```
is what's in master, so not introduced in this commit.

Seems a few lines down it just lists "help" as the command.